code/clue.js

```diff
@@ -1,59 +1,265 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
 const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  firstName: "Jacob",
+  lastName: "Green",
+  color: "green",
+  description: "He has a lot of connections",
   age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  image: "assets/green.png",
+  occupation: "Entrepreneur"
+};
+
+const professorPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "Values knowledge over looks",
+  age: 64,
+  image: "assets/plum.png",
+  occupation: "Professor"
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "Has lips as red as roses",
+  age: 32,
+  image: "assets/scarlet.png",
+  occupation: "Florist"
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She brings her binoculars with her everywhere",
+  age: 40,
+  image: "assets/peacock.png",
+  occupation: "Ornithologist"
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "Goes to a barber every week to trim his beard",
+  age: 71,
+  image: "assets/mustard.png",
+  occupation: "Condiment expert"
+};
+
+const mrsWhite = {
+  firstName: "Skyler",
+  lastName: "White",
+  color: "white",
+  description: "Has at least one coffee stain on her white blouse at all times",
+  age: 45,
+  image: "assets/white.png",
+  occupation: "Accountant"
+};
```

Comment:
You have a lot of information about each person. Maybe that can be outputted in the cards as well?